src/aind_slims_api/core.py

```diff
@@ -204,6 +99,31 @@ def fetch(
 
         return records
 
+    def resolve_model_alias(
+        self,
+        model: Type[SlimsBaseModelTypeVar],
+        attr_name: str,
+    ) -> str:
+        """Given a SlimsBaseModel object, resolve its pk to the actual value"""
+        for field_name, field_info in model.model_fields.items():
+            if field_name == attr_name and field_info.alias:
+                return field_info.alias
+        else:
+            raise ValueError(f"Cannot resolve alias for {attr_name} on {model}")
+
+    def _validate_models(
```

Comment:
This method can also be made static

Reply:
resolved in [c353219](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/20/commits/c353219a49206068eefaad25f4f75c3fc9cc22e9)